models/comments.js

```diff
@@ -18,22 +18,32 @@ exports.fetchComments = async (
       FROM comments
       WHERE article_id = $1 
       ORDER BY ${sort_by} ${order}`;
-    try {
-      const { rows: comments } = await db.query(query, [article_id]);
-      if (comments.length === 0) {
-        return Promise.reject({ status: 404, msg: "Article not found" });
-      }
-      return comments.map((comment) => {
-        return {
-          comment_id: comment.comment_id,
-          votes: comment.votes,
-          created_at: comment.created_at,
-          author: comment.author,
-          body: comment.body,
-          article_id: comment.article_id,
-        };
-      });
-    } catch (err) {
-      return Promise.reject(err);
+  try {
+    const { rows: comments } = await db.query(query, [article_id]);
+    if (comments.length === 0) {
+      return Promise.reject({ status: 404, msg: "Article not found" });
     }
-  };
\ No newline at end of file
+    return comments.map((comment) => {
+      return {
+        comment_id: comment.comment_id,
+        votes: comment.votes,
+        created_at: comment.created_at,
+        author: comment.author,
+        body: comment.body,
+        article_id: comment.article_id,
+      };
```

Comment:
Not sure what this map does here, seems like it doesn't really change anything rather than copy the comment objects